packages/react/src/components/Authenticator/Router/index.tsx

```diff
@@ -11,15 +11,17 @@ import { ConfirmVerifyUser, VerifyUser } from '../VerifyUser';
 import { ConfirmSignIn } from '../ConfirmSignIn/ConfirmSignIn';
 import { ConfirmResetPassword, ResetPassword } from '../ResetPassword';
 
+export type AuthenticatorRenderProp = ({
```

Comment:
Unless there is an absolute need to export this, can we keep it localized to this file?

Reply:
Agreed,

```suggestion
type AuthenticatorRenderProp = ({
```